modules/pel/peltool/peltool.py

```diff
@@ -421,13 +450,19 @@ def main():
 
         sys.exit(0)
 
+    if not inBMC:
```

Comment:
What about the following logic? This way, we can bypass the need to check all options. If any option doesn't anticipate a path, it might be considered an exceptional case for this condition check.

```
if not inBMC:
    if not args.path:
        sys.exit("Outside the BMC environment, please provide the path to the PELs directory using the -P option.")
    if not os.path.isdir(args.path):
       sys.exit(f"{args.path} is not a valid directory")
    PELsPath = args.path
```